### PR TITLE
contrib/v4l2loopback: new package (0.13.2)

### PR DESCRIPTION
**Pull request by subnut**

### CODE REVIEW

File: contrib/v4l2loopback/template.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+@subpackage("v4l2loopback-ckms")
+def _ckms(self):
+    self.pkgdesc = f"{pkgdesc} (kernel sources)"
+    self.install_if = [f"{pkgname}={pkgver}-r{pkgrel}", "ckms"]
+    self.depends = [
+        f"{pkgname}={pkgver}-r{pkgrel}",
+        "ckms",
+        "gmake",
+        "linux-headers",
```
Comment (nekopsykose) [inline contrib/v4l2loopback/template.py:48]:
```suggestion
```

**Comment by nekopsykose** [comment]
you probably want the udev rule too, but rather than group=video you can make it uaccess i guess

**Comment by subnut** [comment]
idk about the uaccess stuff tbh, it seems to be working fine in OBS (after having done modprobe ofc)

Here's the output of `udevadm monitor` when running `modprobe v4l2loopback` -  
(NOTE: this is **without** the udev rule distributed with v4l2loopback)
```
monitor will print the received events for:
UDEV - the event which udev sends out after rule processing
KERNEL - the kernel uevent

KERNEL[34483.779310] add      /devices/virtual/misc/v4l2loopback (misc)
ACTION=add
DEVPATH=/devices/virtual/misc/v4l2loopback
SUBSYSTEM=misc
DEVNAME=/dev/v4l2loopback
SEQNUM=4401
MAJOR=10
MINOR=121

KERNEL[34483.779367] add      /devices/virtual/video4linux/video2 (video4linux)
ACTION=add
DEVPATH=/devices/virtual/video4linux/video2
SUBSYSTEM=video4linux
DEVNAME=/dev/video2
SEQNUM=4402
MAJOR=81
MINOR=2

KERNEL[34483.779397] add      /module/v4l2loopback (module)
ACTION=add
DEVPATH=/module/v4l2loopback
SUBSYSTEM=module
SEQNUM=4403

UDEV  [34483.781261] add      /devices/virtual/misc/v4l2loopback (misc)
ACTION=add
DEVPATH=/devices/virtual/misc/v4l2loopback
SUBSYSTEM=misc
DEVNAME=/dev/v4l2loopback
SEQNUM=4401
USEC_INITIALIZED=34483779238
MAJOR=10
MINOR=121

UDEV  [34483.781648] add      /module/v4l2loopback (module)
ACTION=add
DEVPATH=/module/v4l2loopback
SUBSYSTEM=module
SEQNUM=4403
USEC_INITIALIZED=34483781036

UDEV  [34483.784335] add      /devices/virtual/video4linux/video2 (video4linux)
ACTION=add
DEVPATH=/devices/virtual/video4linux/video2
SUBSYSTEM=video4linux
DEVNAME=/dev/video2
SEQNUM=4402
USEC_INITIALIZED=34483780778
ID_V4L_VERSION=2
ID_V4L_PRODUCT=Dummy video device (0x0000)
ID_V4L_CAPABILITIES=:capture:video_output:
COLORD_DEVICE=1
COLORD_KIND=camera
MAJOR=81
MINOR=2
TAGS=:uaccess:seat:
CURRENT_TAGS=:uaccess:seat:
```

The `uaccess` tag seems to be already set?  
If so, do we even need the included udev rule?

idk really.

**Comment by subnut** [comment]
let me know what to do.
- whether i should include the udev rule or not
- if included, does the udev rule need to be patched to use uaccess instead of the `video` group?

**Comment by nekopsykose** [comment]
sure, seems that it is

**Comment by subnut** [comment]
It's coming from `70-uaccess.rules`, line 34
```
SUBSYSTEM=="video4linux", TAG+="uaccess"
```

File: contrib/v4l2loopback/template.py (hunk 1 above)
Comment (subnut) [reply contrib/v4l2loopback/template.py]: done.

**Pull request merged**